src/app/screens/ledger/verifyLedgerAccountAddress/index.tsx

```diff
@@ -284,6 +288,24 @@ function VerifyLedger(): JSX.Element {
     setCurrentStepIndex(0);
   };
 
+  const getTitle = () => {
+    if (isBitcoinSelected) return t('BTC_ADDRESS');
+    if (isOrdinalSelected) return t('ORDINALS_ADDRESS');
+    if (isStacksSelected) return t('STACKS_ADDRESS');
+
+    return '';
+  };
+
+  const getIconSrc = () => {
+    if (isBitcoinSelected) return btcIcon;
+    if (isOrdinalSelected) return ordinalsIcon;
+    if (isStacksSelected) return stxIcon;
+  };
```

Comment:
why not follow this pattern for all the rest of the render content?

Reply:
Yeah I think it will be improved as the refactoring is being done